### DISCUSSION TITLE
Locate a picture on the map

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#3618] [Q&A]
I was wondering if it was possible to see a picture on the map.
I mean, starting from displayed picture, to have the capability to see its location it on the map and then go back to picture ?

Thanks in advance for your hep.

**Comment by graciousgrey** [comment]
What you can do at the moment to see a photo from the search on the map is clicking on the location displayed in the card view. 
   https://docs.photoprism.app/user-guide/organize/places/

Additionally we do have a related feature request:
- https://github.com/photoprism/photoprism/issues/797
